src/gradcam.jl

```diff
@@ -0,0 +1,20 @@
+"""
+    GradCAM
+
+Analyze model by calculating the gradient of a neuron activation with respect to the input.
+This gradient is then used to calculate the Class Activation Map.
```

Comment:
It would also be great to add a reference to the original paper to the doc string. An example for this can be found in `gradient.jl` and `bibliography.jl`.

Reply:
An entry to the `bibliography.jl` and the addition of the Reference within the docstring of `gradcam.jl` have been added in the commit 26ea2353a697baa4c2b0893cd8054aa4ae988b84.